### ISSUE TITLE
Existance of js script causes silent crash

### ISSUE THREAD

**Issue by Hydrometry** [stax76/mpv.net#270]
**Describe the bug**
Having any .js file in %appdata%\mpv.net\scripts\ causes mpv.net to crash instantly on startup. No messages in console if I try launching from a terminal.

**To Reproduce**
1. open %appdata%\mpv.net\scripts\
2. put any mpv js script there, or create empty .js file
3. try launching mpv.net

**Additional context**
Windows 10 version 2004
mpv.net 5.4.9.0
I tried with fresh mpv install and a fresh config folder. also I tried with [this script ](https://github.com/Arieleg/mpv-copyTime/blob/master/copyTime.js) and an empty .js file as mentioned.

Cheers

**Comment by hooke007** [comment]
It was caused by upstream and has been fixed since libmpv 20210509.
You can try new core [here](https://sourceforge.net/projects/mpv-player-windows/files/libmpv/) and fix it manually.

**Comment by Hydrometry** [comment]
Indeed. Thanks.

**Comment by stax76** [comment]
Yes it was fixed, here is the thread:

https://github.com/mpv-player/mpv/issues/8667

I also tried to find the cause and tried very hard to build libmpv with media auto build suite (mabs) but couldn't really make it, building libmpv for windows isn't easy at all.

My workaround was not including and loading any scripts from the startup folder, but only from the conf folder, and I had converted a script from js to lua.

The latest mpv.net beta has it fixed, so I close this issue.
